packages/devtools_app/lib/src/screens/vm_developer/vm_library_display.dart

```diff
@@ -14,22 +16,35 @@ import 'vm_object_model.dart';
 /// related to library objects in the Dart VM.
 class VmLibraryDisplay extends StatelessWidget {
   const VmLibraryDisplay({
-    required this.library,
     required this.controller,
+    required this.library,
   });
 
-  final LibraryObject library;
   final ObjectInspectorViewController controller;
+  final LibraryObject library;
 
   @override
   Widget build(BuildContext context) {
     final dependencies = library.obj.dependencies;
-    return VmObjectDisplayBasicLayout(
-      object: library,
-      generalDataRows: _libraryDataRows(library),
-      expandableWidgets: [
-        if (dependencies != null)
-          LibraryDependencies(dependencies: dependencies)
+    return Split(
+      axis: Axis.vertical,
+      initialFractions: const [0.5, 0.5],
+      children: [
+        OutlineDecoration.onlyBottom(
+          child: VmObjectDisplayBasicLayout(
+            object: library,
+            generalDataRows: _libraryDataRows(library),
+            expandableWidgets: [
+              if (dependencies != null)
+                LibraryDependencies(dependencies: dependencies)
+            ],
+          ),
+        ),
+        ObjectInspectorCodeView(
+          codeViewController: controller.codeViewController,
+          script: library.scriptRef!,
+          object: library.obj,
+        ),
       ],
```

Comment:
this code seems to be duplicated for each object type. can we create a helper widget that creates the Split and the ObjectInspectorCodeView and takes a parameter for the first child in the split widget?